backend/src/main/java/com/bakdata/conquery/apiv1/StoredQueriesProcessor.java

```diff
@@ -118,17 +136,22 @@ public void deleteQuery(ManagedExecution<?> execution, User user) {
 		storage.removeExecution(execution.getId());
 	}
 
-	public FullExecutionStatus getQueryFullStatus(ManagedExecution query, User user, UriBuilder url) {
+	public FullExecutionStatus getQueryFullStatus(ManagedExecution query, User user, UriBuilder url, Boolean allProviders) {
 
 		user.authorize(query, Ability.READ);
 
 		query.initExecutable(datasetRegistry, config);
 
 		Map<DatasetId, Set<Ability>> datasetAbilities = buildDatasetAbilityMap(user, datasetRegistry);
-		return query.buildStatusFull(storage, url, user, datasetRegistry, datasetAbilities);
+		final FullExecutionStatus status = query.buildStatusFull(storage, url, user, datasetRegistry, datasetAbilities);
+
+		if (query.isReadyToDownload(datasetAbilities)){
```

Comment:
die Logik hast du jetzt doppelt, wäre es nicht sinnvoll die zu inlinen in buildStatus*?

Reply:
Habe die Prozessoren vereinigt